linux_os/guide/system/accounts/accounts-session/accounts_user_dot_group_ownership/oval/shared.xml

```diff
@@ -10,12 +10,18 @@
   <unix:password_object id="object_accounts_user_dot_group_ownership_objects" version="1">
     <unix:username datatype="string" operation="not equal">nobody</unix:username>
     <filter action="include">state_accounts_user_dot_group_ownership_interactive_gids</filter>
+    <filter action="exclude">state_accounts_user_dot_group_ownership_nobody</filter>
```

Comment:
The `<unix:username datatype="string" operation="not equal">nobody</unix:username>` line should do the trick. If the nobody string is not appropriate here, maybe we could check the `gid` field in this object.

Reply:
Also this object only has the username and a filter, hence why I added such filter to avoid matching nobody group.
Unless I'm looking at the wrong spec file version:
https://oval.mitre.org/language/version5.11/ovaldefinition/documentation/unix-definitions-schema.html

Reply:
Ohhh, that is true. It is part of the `password_state`. Nevermind. Sorry for the confusion.